.github/workflows/pypi_publish.yml

```diff
@@ -48,8 +59,122 @@ jobs:
+        echo "LIBRARY_PATH=$(brew --prefix)/lib:$LIBRARY_PATH" >> $GITHUB_ENV
+        echo "CPATH=$(brew --prefix)/include:$CPATH" >> $GITHUB_ENV
+        echo "CPATH=$(python -c 'import sysconfig; print(sysconfig.get_path("include"))'):$CPATH" >> $GITHUB_ENV
+        echo "MACOSX_DEPLOYMENT_TARGET=14.0" >> $GITHUB_ENV
+    - name: Run cibw_before_all_macos
+      if: matrix.os == 'macos-latest'
+      run: |
+        bash ./ci/cibw_before_all_macos.sh "${{ github.workspace }}"
+
+    # Windows environment setup
+    - name: Set up nuget
+      if: matrix.os == 'windows-latest'
+      uses: nuget/setup-nuget@v2
+    - name: Run cibw_before_all_windows
+      if: matrix.os == 'windows-latest'
+      run: |
+        bash ./ci/cibw_before_all_windows.sh "${{ github.workspace }}"
+    - name: Print environment info
+      id: envdata
+      if: matrix.os == 'windows-latest'
+      shell: bash
+      run: |
+        echo $PATH
+        echo $PKG_CONFIG_PATH
+
+        # Inject MSMPI and zlib library paths into github context so we can pass
+        # it to meson; replace any backslashes with forward slashes
+        echo "MSMPI_PATH=$(echo $MSMPI_ROOT | sed 's/\\/\//g')/Lib/x64" >> $GITHUB_OUTPUT
+        echo "ZLIB_PATH=$(echo $ZLIB_ROOT | sed 's/\\/\//g')/lib_release" >> $GITHUB_OUTPUT
+
+    # Build the wheels
+    - name: Triage the build
+      run: bash ./ci/triage_build.sh "${{ matrix.arch }}" "${{ github.event.pull_request.head.sha || github.sha }}" "${{ matrix.python-version }}"
+    - name: Run cibuildwheel
+      uses: pypa/cibuildwheel@v2.22.0
+      env:
+        # Note that tool.cibuildwheel "skip" in pyproject.toml also limits what gets built
+        CIBW_ARCHS: ${{ matrix.arch }}
+        CIBW_BUILD: ${{ env.CIBW_BUILD }}
+        CIBW_SKIP: ${{ env.CIBW_SKIP }}
+        CIBW_BEFORE_BUILD: ${{ env.CIBW_BEFORE_BUILD }}
+        CIBW_BUILD_FRONTEND: ${{ env.CIBW_BUILD_FRONTEND }}
+        CIBW_PRERELEASE_PYTHONS: ${{ env.CIBW_PRERELEASE_PYTHONS }}
+        CIBW_BEFORE_BUILD_LINUX: yum install -y openmpi openmpi-devel zlib-devel; rpm -ql openmpi-devel
+        CIBW_ENVIRONMENT_LINUX: PATH=/usr/lib64/openmpi/bin:$PATH # Make sure mpicc is on PATH
+        CIBW_ENVIRONMENT_MACOS: CPATH=$HDF5_DIR/include:$CPATH
+        CIBW_ENVIRONMENT_WINDOWS: >
+          CPATH="$HDF5_DIR/include;$MSMPI_ROOT/Include;$MSMPI_ROOT/Include/x64;$CPATH"
+          LIBRARY_PATH="$HDF5_DIR/lib;$MSMPI_ROOT/Lib/x64;$LIBRARY_PATH"
+          PATH="$PKGCONFBIN;$PATH"
+
+        # Must use paths relative to the source root here: meson rejects absolute paths
+        CIBW_CONFIG_SETTINGS_WINDOWS: >
+          setup-args="--vsenv"
+          setup-args="-Dmsmpi_libs=['${{ steps.envdata.outputs.MSMPI_PATH }}']"
+          setup-args="-Dzlib_libs=['${{ steps.envdata.outputs.ZLIB_PATH }}']"
+
+    - name: Bundle the hdf5 dlls into the wheels on windows
+      if: runner.os == 'Windows'
+      run: python ./ci/bundle_hdf5_whl.py wheelhouse
```

Comment:
I expect this is going to break things. These DLLs are already in the h5py wheels:
```bash
$ unzip -l h5py-3.13.0-cp313-cp313-win_amd64.whl | grep -v "\.py"
Archive:  h5py-3.13.0-cp313-cp313-win_amd64.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
  3330560  2025-02-14 02:34   h5py/hdf5.dll
   121344  2025-02-14 02:34   h5py/hdf5_hl.dll
    86016  2017-10-04 10:30   h5py/zlib.dll
```

Reply:
Aha, this actually seems like it could be problematic. Let me disable this bundling and see what happens.